src/styles/blocks/main.scss

```
@@ -0,0 +1,64 @@
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  gap: $gap;
+  flex-wrap: wrap;
+  padding: $padding;
+  max-width: calc(7 * $day-size + $padding);
+}
+
+@each $start in $start-days {
+  $margin-left: 0;
+
+  @each $day in $start-days {
+    .calendar--day-start-#{$day} .calendar__day:nth-child(1) {
+      margin-left: $margin-left;
+    }
+
+    $margin-left: $margin-left + $day-size + $gap;
+  }
+}
```

update your `start-days` variable to this one
```
$week: (
  0: "mon",
  1: "tue",
  2: "wed",
  3: "thu",
  4: "fri",
  5: "sat",
  6: "sun");
```
```suggestion
  @each $count, $day in $week {
    &--start-day-#{$day} :nth-child(1) {
      margin-left: $day-size * $count + $gap * $count;
    }
  }
```